tests/integration/internal/tests/api/sandboxes/sandbox_auto_pause_test.go

```diff
@@ -72,14 +72,20 @@ func TestSandboxAutoPauseResumePersisted(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, resp.Body.Close())
 
-	require.Eventually(t, func() bool {
-		res, err := c.GetSandboxesSandboxIDWithResponse(t.Context(), sbxId, setup.WithAPIKey())
-		require.NoError(t, err)
-		require.Equal(t, http.StatusOK, res.StatusCode())
-		require.NotNil(t, res.JSON200)
-
-		return res.JSON200.State == api.Paused
-	}, 10*time.Second, 10*time.Millisecond, "Sandbox is not paused")
+	ctx := t.Context()
+	require.EventuallyWithT(t, func(t *assert.CollectT) {
```

Comment:
this changes the test. It shouldn't ever return an error or any response other than 200. If it does, it should fail